Source/relay/TourGuide/Sets/Fax.ash

```diff
@@ -200,40 +227,42 @@ string [int] SFaxGeneratePotentialFaxes(boolean suggest_less_powerful_faxes, boo
                     
                     description += HTMLGenerateIndentedText(l);
                 }
-                potential_faxes.listAppend(description);
+                add_fax(new potential_fax($monster[gaudy pirate], description));
```

Comment:
can be removed as this is just not viable anymore

Reply:
This is gone now.